elastic/kibana
# [Maps] fix Filters applied to map visualization not preserved when added to dashboard

Fixes https://github.com/elastic/kibana/issues/91282

To test:
1) create map with query and/or filter. Save map
2) Add map to dashboard. Verify map is narrowed by query and/or filter saved with map.

<img width="600" alt="Screen Shot 2022-08-15 at 11 59 47 AM" src="https://user-images.githubusercontent.com/373691/184689975-b8097e3b-7719-4ef2-a29b-e5fc26a349ab.png">

<img width="600" alt="Screen Shot 2022-08-15 at 11 59 37 AM" src="https://user-images.githubusercontent.com/373691/184689996-6817d42e-9f60-487f-836a-15f2749eb8b1.png">
